src/doc/style-guide/src/expressions.md

```diff
@@ -416,13 +440,13 @@ let cstr = "Hi\0" as *const str as *const [u8] as *const std::os::raw::c_char;
 
 ### Chains of fields and method calls
 
-A chain is a sequence of field accesses and/or method calls. A chain may also
-include the try operator ('?'). E.g., `a.b.c().d` or `foo?.bar().baz?`.
+A chain is a sequence of field accesses, method calls, and/or uses of the try
+operator `?`. E.g., `a.b.c().d` or `foo?.bar().baz?`.
 
 Prefer formatting on one line if possible, and the chain is *small*. If
```

Comment:
Should we drop "prefer" here too in the spirit of the language clean up?

Perhaps something like
```suggestion
Format the chain one line if it is *small* and otherwise possible to do so. If
```

Reply:
I kept some of the "prefer" where they seemed to represent "here's how to choose among multiple possible sources of line breaks", but this one I agree is unconditional.